features/proxy_config.feature

```diff
@@ -26,3 +26,40 @@ Feature: Attach command when proxy is configured
            | xenial  |
            | bionic  |
            | focal   |
+
+    @series.lts
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attach command when proxy is configured
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I launch a `focal` `proxy` machine
+        And I run `apt install squid -y` `with sudo` on the `proxy` machine
+        And I add this text on `/etc/squid/squid.conf` on `proxy` above `http_access deny all`:
+            """
+            acl all src 0.0.0.0\/0\nhttp_access allow all
+            """
+        And I run `systemctl restart squid.service` `with sudo` on the `proxy` machine
+        And I configure uaclient `http` proxy to use `proxy` machine
+        And I configure uaclient `https` proxy to use `proxy` machine
+        And I verify `/var/log/squid/access.log` is empty on `proxy` machine
+        And I attach `contract_token` with sudo
+        And I run `canonical-livepatch refresh` with sudo
+        And I run `sleep 120` as non-root
```

Comment:
I wonder if we can set something like this to avoid the 2 minute timeout
```bash
  check-interval: -1  # manually refresh and then run `sudo canonical-livepatch refresh`?
  
```

I've asked in ~CS channel to see if they have a suggestion

Reply:
It seems that using this approach here `sudo canonical-livepatch config check-interval=0; sudo canonical-livepatch refresh` still doesn't remove the delay on the squid logs :(

Reply:
bummer. ok thanks for trying it out.